apps/frontend/src/modules/chat/slices/chat.slice.ts

```diff
@@ -107,6 +108,40 @@ const { actions, name, reducer } = createSlice({
 			})
 			.addCase(getTasksForCategories.rejected, (state) => {
 				state.dataStatus = DataStatus.REJECTED;
+			})
+
+			.addCase(changeTasksSuggestion.pending, (state) => {
+				state.dataStatus = DataStatus.PENDING;
+			})
+			.addCase(changeTasksSuggestion.fulfilled, (state, action) => {
+				state.dataStatus = DataStatus.FULFILLED;
+				const newMessages = action.payload.messages;
+				state.taskSuggestions = [];
+				const newTasks: TaskMessage[] = [];
+
+				for (const message of newMessages) {
+					if (checkIsTaskMessage(message)) {
+						newTasks.push(message.payload);
+						state.taskSuggestions.push({
+							categoryId: message.payload.task.categoryId,
+							categoryName: message.payload.task.categoryName,
+							description: message.payload.task.description,
+							label: message.payload.task.label,
+						});
+					}
+				}
+
+				state.messages.push({
```

Comment:
Can attempt to create a helper function for these if we have time

Reply:
Check the implementation on mobile, @KotenkoNata already created a helper